Sources/PackageModel/UserToolchain.swift

```diff
@@ -142,7 +143,12 @@ public final class UserToolchain: Toolchain {
         {
           return librarian.basename
         }
-        // TODO(5719) use `lld-link` if the build requests lld.
+        // TODO(5719) handle `-Xmanifest` vs `-Xswiftc`
```

Comment:
is this comment relevant?

Reply:
Yes, it is, we still need to refine it further for more ways that the flags can be specified.